src/codegen/TypescriptGenerator.ts

```diff
@@ -370,7 +371,7 @@ ${fieldDefinition}
 }`
   }
 
-  renderDescription(description: Maybe<string>) {
+  renderDescription(description?: Maybe<string>) {
```

Comment:
why `?`
if 
```
type Maybe<T> = null | undefined | T
``` 
[docs](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/graphql/tsutils/Maybe.d.ts)

Reply:
Oops, I accepted the wrong part on merge conflict, you are right. Fixing.